mne/report/report.py

```diff
@@ -3175,7 +3213,7 @@ def _render_epochs(self, *, epochs, add_psd, add_projs, image_format,
             else:
                 fmax = np.inf
 
-            fig = epochs.plot_psd(fmax=fmax, show=False)
+            fig = epochs[epochs_idx].plot_psd(fmax=fmax, show=False)
```

Comment:
Doing this will make a copy of all epochs even if you compute psd on all epochs

Reply:
Excellent catch

Reply:
I've addressed this in d6e6c82

Reply:
Why? A slice is always a view and not a copy.

Reply:
I just saw that `epochs` is not a NumPy array, so I see why it is a copy.